src/solver/solver.jl

```diff
@@ -1,2 +1,9 @@
+module ExitFlag
+  const first_order = "first-order stationary"
+  const max_eval = "maximum number of evaluations"
+  const max_iter = "maximum number of iterations"
+  const max_time = "maximum elapsed time"
+end
```

Comment:
But then does it make sense to create ExitFlag? Doing `status = :stationary` directly would be shorter. Or are you thinking a Dict, like ExitFlag[:stationary]?